applications/openshift/registry/ocp_insecure_registries/rule.yml

```diff
@@ -0,0 +1,57 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: "Check if any insecure registry sources is configured"
+
+description: |-
+    The configuration <tt>registrySources.insecureRegistries</tt> determines the
+    insecure registries that the OpenShift container runtime can access for builds
+    and pods. This configuration setting is for accessing the configured registries
+    without TLS validation which could lead to security breaches and should be
+    avoided.
+
+rationale: |-
+    Insecure registries should not be configured, which would restrict the possibilities of
+    OpenShift container runtime accessing registries which cannot be validated.
+
+severity: medium
+
+identifiers:
+  cce@ocp4: CCE-86023-9
+
+references:
+    nist: CM-7(2),CM-7(4),CM-7(5),CM-11
+    srg: SRG-APP-000014-CTR-000035
+
+ocil_clause: 'insecure registry sources is configured for image access'
+
+ocil: |-
+    <b>Audit:</b>
+
+    To check for the configured insecure registry sources use below command:
+    <tt>$ oc get image.config.openshift.io/cluster -ojsonpath='{.spec.registrySources}'</tt>
+    The output lists the insecure registry sources configured.
+
+    <b>Remediation:</b>
+    To remove configured insecure resgitry sources use below command:
+    <tt>$ oc edit image.config.openshift.io/cluster</tt>
+    Remove any registries listed under insecureRegistries.
+
+    Refer to https://docs.openshift.com/container-platform/latest/openshift_images/image-configuration.html
```

Comment:
btw when you move the link to description, please use the `weblink` macro, e.g.:
```
For more information, follow
    {{{ weblink(link="https://docs.openshift.com/container-platform/latest/openshift_images/image-configuration.html",
                text="the relevant documentation") }}}.
```

Reply:
comment incorporated.